.circleci/config.yml

```diff
@@ -22,16 +19,19 @@ workflows:
           context: Danger
       - kitchen/delivery:
           name: delivery
-
-      # We cannot test using dokken due to local make file
```

Comment:
I don't think we can run these tests in circle ci

Reply:
We don't need to test the plugin with the local make.  The list of plugins installed is mostly arbitrary.